scripts/setup.sh

```diff
@@ -9,7 +9,7 @@ else
   echo "node ${NODE_VERSION} installation failed."
 fi
 
-YARN_VERSION=1.22.17
+YARN_VERSION=3.2.0
```

Comment:
can we update this for all our github actions for CI as well? see here: https://github.com/okta/odyssey/blob/develop/.github/workflows/ci.yml#L15-L19 I'm hoping this setup-node action allows us to configure an explicit yarn version too. We'll need to update that across all our workflow for github actions

Reply:
I'm not sure this github action plugin allows for this: https://github.com/actions/setup-node

I think we may need to write a custom action to install the correct yarn version instead? or perhaps Github has a declarative way to set this up in our configuration YAML

Reply:
Yeah, as far as I can tell, it appears that `yarn` will automatically detect and use whatever version we specify. `.yarnrc.yml` specifies a `yarnPath` which is `.yarn/releases/yarn-3.2.0.cjs`. So we should not need to do any extra work in CI setup. In the Github Action for this PR, it is correctly using yarn 3.2.0 (see "setup node" step where it calls `/usr/local/bin/yarn --version`)